source/whats-new.txt

```diff
@@ -37,6 +38,29 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.6:
+
+What's New in 5.6
+-----------------
+
+The {+driver-short+} v5.6 release includes the following features:
+
+- The driver now supports Node.js v20.
+
+- The driver can return a cursor as the response to a server command by
+  calling the ``runCursorCommand()`` method. To learn more about this feature,
+  see the `runCursorCommand API documentation <{+api+}/5.6/classes/Db.html#command>`__.
+
+- Support for specifying time series collection creation options
+  `bucketMaxSpanSeconds <{+api+}/interfaces/TimeSeriesCollectionOptions.html#bucketMaxSpanSeconds>`__
+  and
+  `bucketRoundingSeconds <{+api}/interfaces/TimeSeriesCollectionOptions.html#bucketRoundingSeconds>`__.
+  To learn more about these time series collection options, see the
+  `Set Granularity for Time Series Data <https://www.mongodb.com/docs/upcoming/core/timeseries/timeseries-granularity/#set-granularity-for-time-series-data>`__
+  Server Manual documentation.
```

Comment:
S: decapitalize manual in this line (I think this is what we agreed to going forwards?)

Reply:
Thanks for the reminder!